### PR TITLE
Art categories 2

### PR DESCRIPTION
**Pull request by Jtpiland**
## Implemented Features:
- Add ArtCategory controller with create method, route and spec 
- Add factory for art and category  

## Roadblocks/Logs/Errors:
-

## Testing:
- Only tests for creation in db, no edge cases

## Notes/For the future:
- Refactor into helper methods 
- Make testing stronger?

### CODE REVIEW

**Review by marlitas — APPROVED** [review]
Looks great! There's some merge conflicts, but down to merge it in one we look at those!

File: app/controllers/api/v1/art_categories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Api::V1::ArtCategoriesController < ApplicationController
```
Comment (deebot10) [inline app/controllers/api/v1/art_categories_controller.rb:1]: Nice work!

File: app/facades/get_art_facade.rb [hunk 2]
```diff
@@ -27,29 +27,22 @@ def art_categories(number, size)
           Category.find_or_create_by(name: description[:description])
         end
       end
+      category.each do |artsy_id, value|
+        value[:responses][0][:labelAnnotations].each do |description|
+          ArtCategory.create(art_id: (Art.find_by(artsy_id: artsy_id).id), category_id: (Category.find_by(name: description[:description]).id))
+        end
+      end
```
Comment (deebot10) [inline app/facades/get_art_facade.rb:34]: Good Stuff!
